File: November/Day11/removeDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
         int n = nums.length;
         if(n < 2) return n;
        int indx = 1;
        for(int i = 1; i < n; i++) 
            if(nums[i] != nums[i-1]) nums[indx++] = nums[i];
        return indx;
    }
}

File: November/Day11/remove_duplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) < 2:
            return 1

        left = 0
        right = 1
        
        while right < len(nums):
            if nums[left] != nums[right]:
                left += 1
                nums[left] = nums[right]
                right += 1
               
            else:
                right += 1
        
        return left + 1
